### DISCUSSION TITLE
#34. VATZ biweekly meeting at 2023-05-24 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#437] [General]
### 1. Overall

<img width="1331" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/90d372dd-1eb1-47e4-aa3f-c91cd122e742">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
32 |  2/17 | 10
33 |  4/20 | 20

---

Schedule
v1.0 
- vatz tag
- cosmoshub tag
- Sysutil tag

Documents 
- installation @qkrybin 
   - upgrade need 

PR 
- https://github.com/dsrvlabs/vatz-plugin-sysutil/pull/9
   - Please review  ASAP
      - @meetrick @gnongs @skonhwang @kim201212  

@heejin-github 
- tag matching for every repo 
   meeting at 5/30/2023 (KST)

@dsrvlabs/validator 
- VATZ upgrade with 1.0  (will create RC-1.0 in 5/26/2023)
   - Please, report to VATZ issue if there's any bugs or error you find.   

---

<img width="1014" alt="image" src="https://github.com/dsrvlabs/vatz/assets/6308023/7630ad85-a1d0-4884-96ab-e3245a2852a0">

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/435
   - https://github.com/dsrvlabs/vatz/pull/443
- https://github.com/dsrvlabs/vatz/issues/415
- https://github.com/dsrvlabs/vatz/issues/444
   - https://github.com/dsrvlabs/vatz/pull/445 
- https://github.com/dsrvlabs/vatz/issues/436

**Comment by rootwarp** [comment]
# 준교
- https://github.com/dsrvlabs/vatz/issues/426
  - https://github.com/dsrvlabs/vatz/pull/433
- https://github.com/dsrvlabs/vatz/issues/439
  - https://github.com/dsrvlabs/vatz/pull/441

**Comment by kim201212** [comment]
# 형기

- https://github.com/dsrvlabs/vatz/issues/344
   - https://github.com/dsrvlabs/vatz/pull/442

**Comment by heejin-github** [comment]
## 희진
- #423 
  - https://github.com/dsrvlabs/vatz-plugin-sysutil/pull/9

**Comment by skonhwang** [comment]
## 승곤
- no issue

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/417

**Comment by xellos00** [comment]
# For reference, versioning. 
- https://go.dev/doc/modules/version-numbers
- https://go.dev/blog/publishing-go-modules

@xellos00 
test VATZ and official plugins version as above.
